src/main/resources/templates/js/trends-chart.js.vm

```diff
@@ -1,20 +1,23 @@
+#macro(stringArray $array) [#foreach($element in $array) "$element", #end] #end
+#macro(numberArray $array) [#foreach($element in $array) $element, #end] #end
+
```

Comment:
I prefer to keep logic in Java because it allows me to test it easily and by having more logic in velocity I'm increasing potential regression like the one with tags because this macro is not tested